oauth2client/client.py

```diff
@@ -1303,7 +1307,7 @@ def _get_well_known_file():
                                         _CLOUDSDK_CONFIG_DIRECTORY)
 
   if not os.path.isdir(default_config_dir):
-    raise OSError('Config directory does not exist', default_config_dir)
+    logging.info('Config directory does not exist: %s', default_config_dir)
```

Comment:
`info` shouldn't be on by default, but you're right that it probably doesn't serve much purpose. dropping for now.